src/main/scala/firrtl/passes/ZeroWidth.scala

```diff
@@ -25,19 +25,12 @@ object ZeroWidth extends Transform with DependencyAPIMigration {
     case _          => false
   }
 
-  private def makeEmptyMemBundle(name: String): Field =
-    Field(
-      name,
-      Flip,
-      BundleType(
-        Seq(
-          Field("addr", Default, UIntType(IntWidth(0))),
-          Field("en", Default, UIntType(IntWidth(0))),
-          Field("clk", Default, UIntType(IntWidth(0))),
-          Field("data", Flip, UIntType(IntWidth(0)))
-        )
-      )
-    )
+  private def makeZero(tpe: ir.Type): ir.Type = tpe match {
+    case ClockType => UIntType(IntWidth(0))
+    case a: UIntType      => a.copy(IntWidth(0))
+    case a: SIntType      => a.copy(IntWidth(0))
+    case a: AggregateType => a.map(makeZero)
+  }
```

Comment:
```suggestion
  private def makeZeroWidth(tpe: ir.Type): ir.Type = tpe match {
    case ClockType => UIntType(IntWidth(0))
    case a: UIntType      => a.copy(IntWidth(0))
    case a: SIntType      => a.copy(IntWidth(0))
    case a: AggregateType => a.map(makeZeroWidth)
  }
```

I was about to link you to `Utils.getGroundZero` because that's the functionality that `makeZero` implies to me

Reply:
Well I'm too late, not worth changing heh